cloud_launch.json

```diff
@@ -13,6 +13,10 @@
       {
         "name": "interest_queries_components_are_filtered_by_component_delivery",
         "value": "false"
+      },
+      {
+        "name": "enforce_read_acl_in_qbi",
+        "value": "false"
```

Comment:
What does this do? Is this a publicly documented flag?

Reply:
It stops acl read access from overruling the interest defined in qbi

aka acl is for write-access only, interest is for read-access only

Reply:
Right.. are we recommending users add this to their launch config too?

Reply:
I think it would make sense to tell users to add this if it's documented (talking to runtime to figure out if/when that's going to happen)

Reply:
removed it, just set read access to all workers